<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openai</class>
 <widget class="QDialog" name="openai">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Completion tokens</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Enable API</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="apiBase_lineEdit">
        <property name="toolTip">
         <string extracomment="IP of your model. Leave blank if you are not hosting any model and planning on using OpenAI services."/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="temperature_lineEdit">
        <property name="text">
         <string>0.7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>API key</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>API base</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>API type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="modelName_lineEdit">
        <property name="toolTip">
         <string extracomment="Test"/>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="text">
         <string>gpt-3.5-turbo</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="completionTokens_lineEdit">
        <property name="text">
         <string>200</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="apiKey_lineEdit"/>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="copyKey_pushButton">
        <property name="text">
         <string>Copy key from ENV</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
